Create template files for documentation (#2)
ansible-community/project-template: docs/references/roadmap.md

# PROJECTNAME $version Roadmap

> note to developers - this should be a bullet list or other simple list of the
>  features or changes expected in the next release.

Reviewer: ```suggestion
Roadmaps should be a bulleted, or other simple, list of features or changes expected in the next release.
```
